### PR TITLE
datapages schema updates

### PR DESCRIPTION
**Pull request by mlbrgl**

### CODE REVIEW

**Comment by mlbrgl** [comment]
Current dependencies on/for this PR:
* master
  * **PR #2218** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2218" target="_blank">Graphite</a> 
    * **PR #2233** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2233" target="_blank">Graphite</a> 
      * **PR #2248** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2248" target="_blank">Graphite</a> 
        * **PR #2250** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2250" target="_blank">Graphite</a> 
          * **PR #2259** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2259" target="_blank">Graphite</a> 
            * **PR #2270** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2270" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/owid/owid-grapher/2270?utm_source=stack-comment).

File: packages/@ourworldindata/utils/src/owidTypes.ts [hunk 1]
```diff
@@ -1191,12 +1191,15 @@ export const AllowedDataPageGdocFields = [
...
     // This is a hacky way of handling sourceDescription fields, entered in the
-    // gdoc as sourceDescription1, sourceDescription2, etc... The 50 limit is
+    // gdoc as sourceDescription1, sourceDescription2, etc... The 10 limit is
     // arbitrary and should be plenty, but could be increased if needed.
-    ...[...Array(50)].map((_, idx) => `sourceDescription${idx + 1}`),
+
+    // EDIT: this actually whitelists all strings in typechecking, making it
+    // irrelevant. As it is, this only serves as documentation but not as
+    // typechecking.
+    ...[...Array(10)].map((_, idx) => `sourceDescription${idx + 1}`),
```
Comment (ikesau) [inline packages/@ourworldindata/utils/src/owidTypes.ts:1202]: At this point I'd just write out

```ts
  "sourceDescription1",
  "sourceDescription2",
  "sourceDescription3",
  "sourceDescription4",
  "sourceDescription5",
  "sourceDescription6",
  "sourceDescription7",
  "sourceDescription8",
  "sourceDescription9",
  "sourceDescription10"
```

**Review by ikesau — APPROVED** [review]

**Comment by mlbrgl** [comment]
[Graphite](https://app.graphite.dev/github/pr/owid/owid-grapher/2270) rebased this pull request as part of a merge.

**Comment by mlbrgl** [comment]
@mlbrgl merged this pull request with [Graphite](https://app.graphite.dev/github/pr/owid/owid-grapher/2270).

**Pull request merged**